libs/@guardian/react-crossword/src/hooks/useProgress.ts

```diff
@@ -109,3 +109,5 @@ export const useProgress = (data: CAPICrossword, userProgress?: Progress) => {
 		isPersistent,
 	] as const;
 };
+
+export type UseProgress = ReturnType<typeof useProgress>;
```

Comment:
How do you feel about making this an object. I only say that because the type UseProgress[1] seems less descriptive than UseProgress['setProgress']. I see that this is more similar to useState etc.

Reply:
i had that same thought yeah, i think it's a good idea

Reply:
done in 49e1718